Appwrite/Appwrite_User_Authentication.ipynb

```diff
@@ -0,0 +1,390 @@
+   },
+   "source": [
+    "# Appwrite - User Authentication"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf188fc8-9ea3-4853-ae06-06c3fe162226",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Tags:** #appwrite #python #sdk #authentication #user #create"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f122d69a-ae50-4d95-90df-fcb0d483fac7",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Author:** [SaiKiran M](www.linkedin.com/in/msaikiran9)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f8a21c1-560a-405f-8599-fd906e95673d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Last update:** 2023-10-08 (Created: 2023-10-08)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96df94c1-842e-4588-a304-7b28a281138e",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Description:** This notebook will demonstrate how to use the Appwrite Python SDK to create a authentication and have options to `create_user`, `delete_user`, `list_all_users` as individual routines. It is useful for organizations that need to manage user authentication."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a14df074-778c-4f39-bc9e-c53d907dc830",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**References:**\n",
+    "- [Appwrite Python SDK](https://github.com/appwrite/sdk-for-python)\n",
+    "- [Appwrite Documentation](https://appwrite.io/docs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2bfe3fda-6f51-45b4-8dfa-06dda53622b2",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f245a0a-f3a4-4750-be9c-752a07e7e4b5",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "81730041-2523-462e-83c7-f3c08a3ca960",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from appwrite.client import Client\n",
+    "from appwrite.services.users import Users"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "54602e57-1b2f-4acb-9f32-eb5ec0561582",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Setup variables\n",
+    "- `endpoint`: Appwrite endpoint URL\n",
+    "- `project`: Appwrite project ID\n",
+    "- `key`: Appwrite secret key"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "69019e17-53eb-434a-8837-ab33b6a5d0ce",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "endpoint = \"https://cloud.appwrite.io/v1\" #replace with your end-point here.\n",
+    "project = \"6526da963cd09677a562\" #replace with your project here.\n",
```

Comment:
@MSaiKiran9 Make sure to remove the API keys, since these shouldn't be exposed. You can make use of `secret` feature of naas to facilitate the same. Docs can be found [here](https://docs.naas.ai/features/secret).

Reply:
@srini047 I just left it for the preview sake after the merge i'll delete it anyway as it's my private key .